### PR TITLE
Card Dropown, Playlists Modal & Single Playlist Page

### PR DESCRIPTION
**Pull request by horsemaker**

### CODE REVIEW

**Review by vijaytembugade — APPROVED** [review]
Code looks so clean. 
Everything working fine too.
Good to go

File: src/frontend/components/ExpandedSidebar/ExpandedSidebar.js [hunk 1]
```diff
@@ -136,20 +138,21 @@ export const ExpandedSidebar = () => {
...
-        <span className="sidebar-option">Playlists</span>
-      </NavLink>
+      {playlists.map((playlist) => (
+        <NavLink
+          key={playlist._id}
+          to={`/playlists/${playlist._id}`}
+          className={({ isActive }) =>
+            isActive
+              ? "expanded-sidebar-link active-link"
+              : "expanded-sidebar-link"
```
Comment (t007rushi) [inline src/frontend/components/ExpandedSidebar/ExpandedSidebar.js]: extract outside and pass only a single word as per active style convention

File: src/frontend/components/ShrinkedSidebar/ShrinkedSidebar.js [hunk 2]
```diff
@@ -83,21 +86,22 @@ export const ShrinkedSidebar = () => {
...
-          title="Playlists"
-          className="material-icons-outlined sidebar-icon"
+      {playlists.map((playlist) => (
+        <NavLink
+          key={playlist._id}
+          to={`/playlists/${playlist._id}`}
+          className={({ isActive }) =>
+            isActive
+              ? "shrinked-sidebar-link active-link"
+              : "shrinked-sidebar-link"
```
Comment (t007rushi) [inline src/frontend/components/ShrinkedSidebar/ShrinkedSidebar.js]: same here

File: src/frontend/pages/HistoryPage/HistoryPage.js [hunk 3]
```diff
@@ -25,11 +25,13 @@ export const HistoryPage = () => {
         <div className="user-property-page-cover-media">
           <img
             className="user-property-page-cover-img"
-            src={`https://img.youtube.com/vi/${history[0]._id}/maxresdefault.jpg`}
+            src={`https://img.youtube.com/vi/${
+              history[history.length - 1]._id
```
Comment (t007rushi) [inline src/frontend/pages/HistoryPage/HistoryPage.js:29]: Nicely done
this remembers me Stack approach

**Review by t007rushi — APPROVED** [review]
Great UI man 🔥Red-Black is ♦
Original ones Gonna sue you💯

File: src/frontend/components/ExpandedSidebar/ExpandedSidebar.js (hunk 1 above)
Comment (horsemaker) [reply src/frontend/components/ExpandedSidebar/ExpandedSidebar.js]: Right! Thank you.

**Pull request merged**